actors/miner/src/lib.rs

```diff
@@ -3793,9 +3792,9 @@ fn extend_simple_qap_sector(
 
         new_sector.expiration = new_expiration;
         // update deal weights to account for new duration
-        new_sector.deal_weight = deal_space * (new_sector.expiration - new_sector.activation);
+        new_sector.deal_weight = deal_space * (new_sector.expiration - new_sector.power_base_epoch);
         new_sector.verified_deal_weight = BigInt::from(*new_verified_deal_space)
-            * (new_sector.expiration - new_sector.activation);
+            * (new_sector.expiration - new_sector.power_base_epoch);
```

Comment:
The use of `sector.activation` above at 3760, computing `old_duration`, also needs to be changed to `power_base_epoch`. The sector may have been updated before being extended.